### PR TITLE
Find/fix blackhole rescues swallowing exceptions

### PR DESCRIPTION
**Pull request by billy-ran-away**
This is a pretty simple pull request that just finds and fixes blackhole rescues that are swallowing exceptions and not giving us any warning making debugging more difficult.

### CODE REVIEW

File: app/models/forms/broker_agency_profile.rb [hunk 1]
```diff
@@ -169,7 +169,8 @@ def update_attributes(attr)
         person.update_attributes(extract_person_params)
         person.emails.find_by(kind: 'work').update(address: attr[:email])
       end
-    rescue
+    rescue Exception => e
```
[the remark this answers is not in the document]
Comment (TreyE) [reply app/models/forms/broker_agency_profile.rb]: This actually rescues MORE exceptions than the previous code.

File: app/models/caches/plan_details.rb [hunk 2]
```diff
@@ -6,15 +6,16 @@ def self.lookup_rate(plan_id, rate_schedule_date, effective_age)
         (pt[:start_on] <= rate_schedule_date) && (pt[:end_on] >= rate_schedule_date)
       end
       age_record[:cost]
-    rescue
+    rescue Exception => e
```
[the remark this answers is not in the document]
Comment (TreyE) [reply app/models/caches/plan_details.rb]: This actually rescues MORE exceptions than the previous code.

File: app/controllers/employers/census_employees_controller.rb [hunk 3]
```diff
@@ -35,7 +35,8 @@ def create
       begin
         missing_kind = census_employee_params['email_attributes']['kind']==''
         @census_employee.errors['Email']='Kind must be selected' if missing_kind
-      rescue
+      rescue Exception => e
+        Rails.logger.error { "Could not create census_employee with #{census_employee_params} because of #{e}" }
```
[the remark this answers is not in the document]
Comment (TreyE) [reply app/controllers/employers/census_employees_controller.rb]: This actually rescues MORE exceptions than the previous code.

File: app/controllers/employers/census_employees_controller.rb [hunk 4]
```diff
@@ -35,7 +35,8 @@ def create
       begin
         missing_kind = census_employee_params['email_attributes']['kind']==''
         @census_employee.errors['Email']='Kind must be selected' if missing_kind
-      rescue
+      rescue Exception => e
```
[the remark this answers is not in the document]
Comment (TreyE) [reply app/controllers/employers/census_employees_controller.rb]: This actually rescues MORE exceptions than the previous code.

File: app/models/aws/s3_storage.rb [hunk 5]
```diff
@@ -19,6 +21,7 @@ def save(file_path, bucket_name, key=SecureRandom.uuid)
           nil
         end
       rescue Exception => e
```
Comment (antonioschaffert) [inline app/models/aws/s3_storage.rb]: Shouldn't we also follow StandardError here?

**Pull request closed without merging**